packages/dd-trace/src/plugins/tracing.js

```diff
@@ -81,6 +82,8 @@ class TracingPlugin extends Plugin {
       }
     })
 
+    if (span && this.component) { span.setTag(COMPONENT, this.component) }
```

Comment:
`span` is guaranteed to have a value and `setTag` handles empty values properly, so the condition is not needed at all.